files/jcl/ZWESECUR.jcl

```diff
@@ -310,133 +347,143 @@ $$
 INSERT STC.&XMEMSTC. LOGONID(&XMEMUSER.) GROUP(&STCGROUP.) +
 STCID(&XMEMSTC.)
 F ACF2,REFRESH(STC)
 *
-* started task for XMEM auxilary cross memory server                *
+* started task for XMEM auxilary cross memory server
 *
 SET CONTROL(GSO)
 INSERT STC.&AUXSTC. LOGONID(&AUXUSER.) GROUP(&STCGROUP.) +
 STCID(&AUXSTC.)
 F ACF2,REFRESH(STC)
 *
-* operator command F ACF2,REFRESH(STC)                              *
+* DEFINE ZOWE SERVER PERMISIONS ...................................
 *
-* DEFINE ZOWE SERVER PERMISIONS ................................... *
+* permit Zowe main server to use XMEM cross memory server
 *
-* permit Zowe main server to use XMEM, cross memory server          *
+SET RESOURCE(FAC)
+RECKEY ZWES ADD(IS SERVICE(READ) ROLE(&STCGROUP.) ALLOW)
+F ACF2,REBUILD(FAC)
+*
+* permit Zowe main server to create a user's security environment
+* ATTENTION: Defining the BPX.DAEMON or BPX.SERVER profile makes
+*            z/OS UNIX switch to z/OS UNIX level security. This is
+*            more secure, but it can impact operation of existing
+*            applications. Test this thoroughly before activating
+*            it on a production system.
 *
 SET RESOURCE(FAC)
-RECKEY ZWES ADD(IS SERVICE(READ) ROLE(&ADMINGRP.) ALLOW)
+RECKEY BPX ADD(SERVER SERVICE(UPDATE) ROLE(&STCGROUP.) ALLOW)
 F ACF2,REBUILD(FAC)
 *
-** comment out to not use SUPERUSER.FILESYS, see JCL comments       *
-** permit Zowe main server to write persistent data                 *
+** comment out to not use SUPERUSER.FILESYS, see JCL comments
+** permit Zowe main server to write persistent data
 *
   SET RESOURCE(UNI)
-  RECKEY SUPERUSER.FILESYS ADD(SERVICE(READ) ROLE(&ADMINGRP.) ALLOW)
+  RECKEY SUPERUSER.FILESYS ADD(SERVICE(READ) ROLE(&STCGROUP.) ALLOW)
   F ACF2,REBUILD(UNI)
 *
-* DEFINE XMEM SERVER PERMISIONS ................................... *
+* DEFINE XMEM SERVER PERMISIONS ...................................
 *
-* permit XMEM to create a user's security environment               *
-* ATTENTION: Defining the BPX.DAEMON or BPX.SERVER profile makes    *
-*            z/OS UNIX switch to z/OS UNIX level security, which is *
-*            more secure, but it can impact operation of existing   *
-*            applications. Test this thoroughly before activating   *
-*            it on a production system.                             *
+* permit XMEM to create a user's security environment
+* ATTENTION: Defining the BPX.DAEMON or BPX.SERVER profile makes
+*            z/OS UNIX switch to z/OS UNIX level security. This is
+*            more secure, but it can impact operation of existing
+*            applications. Test this thoroughly before activating
+*            it on a production system.
 *
 SET RESOURCE(FAC)
-RECKEY BPX ADD(DAEMON SERVICE(UPDATE) ROLE(&ADMINGRP.) ALLOW)
-RECKEY BPX ADD(SERVER SERVICE(UPDATE) ROLE(&ADMINGRP.) ALLOW)
+RECKEY BPX ADD(DAEMON SERVICE(UPDATE) ROLE(&STCGROUP.) ALLOW)
+RECKEY BPX ADD(SERVER SERVICE(UPDATE) ROLE(&STCGROUP.) ALLOW)
```

Comment:
This seems to be a duplicate definition because it was already done in the zowe user section above. The BPX.DAEMON and BPX.SERVER access is defined for the for the ROLE(&STCGROUP). I see that only &ZOWEUSER is inserted for the ROLE. &XMEMUSER is not in the ROLE which seems to be correct

Reply:
yes, this change is to fix a typo, it is not the admins but the started task group that need the permit. These are the same by default, but can differ.